src/dags/stroomstoringen.py

```diff
@@ -0,0 +1,137 @@
+from datetime import datetime
+
+from airflow import DAG
+from airflow.models import Variable
+from airflow.operators.bash_operator import BashOperator
+from airflow.operators.postgres_operator import PostgresOperator
+from common import (
+    DATAPUNT_ENVIRONMENT,
+    SHARED_DIR,
+    MessageOperator,
+    default_args,
+    quote_string,
+    slack_webhook_token,
+)
+from common.db import DatabaseEngine
+from contact_point.callbacks import get_contact_point_on_failure_callback
+from http_fetch_operator import HttpFetchOperator
+from ogr2ogr_operator import Ogr2OgrOperator
+from postgres_permissions_operator import PostgresPermissionsOperator
+from postgres_rename_operator import PostgresTableRenameOperator
+from provenance_rename_operator import ProvenanceRenameOperator
+from sql.stroomstoringen import CONVERT_DATE_TIME
+
+dag_id: str = "stroomstoringen"
+variables: dict = Variable.get("stroomstoringen", deserialize_json=True)
+endpoint_url: str = variables["data_endpoints"]["stroomstoringen"]
+
+
+today = datetime.now().strftime("%m-%d-%Y")
+
+tmp_dir: str = f"{SHARED_DIR}/{dag_id}"
```

Comment:
We are half way between using `Path` versus string. For example the `HttpFetchOperator` already does accept a `Path` object for `tmp_file`. But the `Ogr2OgrOperator` does not. Personally I'd rather see us dealing with `Path` objects for as long as possible and only convert to a string when needed.

But as not all operators support it yet, a string should be okay for now. Though this could be an opportunity to convert the `Ogr2OgrOperator` to accept both `Path` and `str` similarly to `HttpFetchOperator`